snuba/web/query.py

```diff
@@ -201,15 +232,15 @@ def _format_storage_query_and_run(
     timer.mark("prepare_query")
 
     stats = {
-        "clickhouse_table": table_name,
+        "clickhouse_table": table_names,
         "final": from_clause.final,
```

Comment:
I still have to deal with this before this method can actually take in the the composite query.